db2_for_machine_learning_samples/README.md

```diff
@@ -0,0 +1,74 @@
+# Db2 Samples For Machine Learning 
+
+Nowadays data scientists and data engineers have to collect, handle, and analyze huge sets of data. These huge sets of data allow companies and teams to make valuable and educated predictions and decisions for their buisness. However, these data sets are so huge that they won't fit in on your computer, or the script may take a long time to run. Due to this, these datasets are being stored in databases such as Db2 on Cloud that allows companies to hold huge datasets and run models much more quickly and scale at an alarming rate. 
+
+In these three notebooks, we will be demonstrating how programmers can use Db2 as a data source when creating machine learning models with huge data sets. Each notebook talks about how Db2 can be used with a specific popular machine learning libarary- `sklearn`, `h2o`, and `tensorflow`.
+
+Before you start using these notebooks, make sure you have created a Db2 On Cloud instance.
```

Comment:
What if I already have Db2 somewhere else - can these samples be used outside of a Db2 on Cloud instance?  Should we also provide guidance for scenarios where an instance may already exist?

Reply:
Okay, just updated it!